tests/test_fsclient.py

```diff
@@ -42,15 +42,95 @@
 DEFAULT_RC = {}
 rc = DEFAULT_RC
 
+
 # FileSystemClient methods tested here
 def test_is_alive():
     expected = True  #filesystem is always alive!
     fsc = FileSystemClient(rc)
     actual = fsc.is_alive()
     assert actual == expected
 
+
 @pytest.mark.skip("Not written")
 def test_open():
     pass
 
-# and so on.....
+
+@pytest.mark.skip("Not written")
+def test_load_json():
+    pass
+
+
+@pytest.mark.skip("Not written")
+def test_load_yaml():
+    pass
+
+
+@pytest.mark.skip("Not written")
+def test_load_database():
+    pass
+
+
+@pytest.mark.skip("Not written")
+def test_dump_json():
+    pass
+
+
+@pytest.mark.skip("Not written")
+def test_dump_yaml():
+    pass
+
+
+@pytest.mark.skip("Not written")
+def test_dump_database():
+    pass
+
+
+@pytest.mark.skip("Not written")
+def test_close():
+    pass
+
+
+@pytest.mark.skip("Not written")
+def test_keys():
+    pass
+
+
+@pytest.mark.skip("Not written")
```

Comment:
we may not have to test the private functions, htings like getters and setters and constructors.